docs/plainmonthday.md

```diff
@@ -42,6 +46,12 @@ Usage examples:
 md = new Temporal.PlainMonthDay(3, 14); // => 03-14
 // Leap day
 md = new Temporal.PlainMonthDay(2, 29); // => 02-29
+
+// 1 Adar I (first day of a leap month) in the Hebrew calendar
```

Comment:
Maybe add a comment saying "normally don't do this"?

Reply:
Actually, seems safer to just remove it so we won't give people bad ideas. I'll remove it in my next commit.